timelock-server/build.gradle

```diff
@@ -39,11 +45,44 @@ dependencies {
     testCompile group: 'org.mockito', name: 'mockito-core'
     testCompile project(":atlasdb-tests-shared")
 
-    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing'
+    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing', version: libVersions.timelock_dropwizard
 }
 
 distribution {
     serviceName "timelock-server"
     mainClass 'com.palantir.atlasdb.timelock.TimeLockServerLauncher'
     args 'server', 'var/conf/timelock.yml'
+    defaultJvmOpts "-Xms512m",
+            "-Xmx512m",
+            "-javaagent:service/lib/jetty-alpn-agent-${libVersions.jetty_alpn_agent}.jar"
+}
+
+dependencyRecommendations {
```

Comment:
Its overridden for all projects, so we need to override if we want a different resolution by nebula for this particular module.